src/index.js

```diff
@@ -1,17 +1,27 @@
 import 'bootstrap/dist/css/bootstrap.css'
 
 import { Container, Navbar, NavbarBrand } from 'reactstrap'
-import { css, cx } from 'react-emotion'
+import { css, cx, injectGlobal } from 'react-emotion'
 import { faCompass } from '@fortawesome/free-solid-svg-icons'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 
 import React from 'react'
 import ReactDOM from 'react-dom'
 
-import { GRAY_900, BLUE_GRAY_900 } from './assets/colors'
+import { GRAY_900, BLUE_GRAY_900, LIGHT_BLUE } from './assets/colors'
 import FeedbackButton from './components/feedback-button'
 import MortgageInvestmentCompare from './components/mortgage-investment-compare/index'
 
+// GLOBAL STYLES
+injectGlobal`
+  color: ${GRAY_900};
+  font-family: 'Noto Sans', sans-serif;
```

Comment:
Does this work? It's not wrapped under any tag

Reply:
ah I see, `injectGlobal` doesn't work like I thought it would. Let me correct it

Reply:
#45